.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,19 @@
+name: NPM publish
+
+on:
+  release:
```

Comment:
Just wondering as a none JS expect - How does it determine the version?

Reply:
The version is configured in the `package.json` file, if that is what you mean.
But it is a good thing you mentioned this, as we still would need to update that manually or via `npm version`.
I can add this as an extra build step and extract the version out of the release that has been created (via `github.event.release.tag_name`), but not to sure how this will react when the `workflow_dispatch` is running.

Reply:
That would be ideal indeed. With `workflow_dispatch` you can add parameters as well would allow you to do the same thing.

I've recently set this up [here](https://github.com/tomkerkhove/k8s-event-grid-bridge/blob/main/.github/workflows/push-stable-image.yml#L3-L8) for dispatch and [here](https://github.com/tomkerkhove/k8s-event-grid-bridge/blob/main/.github/workflows/push-stable-image.yml#L44-L65) to use it

Reply:
Last commit should do the trick, not sure if the newly created tag will be read properly within the same job, but we'll see when this runs (Bitbucket had this issue that we first needed to pull again before publish).

Reply:
SHould be fine, I think - You never know with these kind of things :D